### PR TITLE
More docker php ports

### PR DESCRIPTION
**Pull request by nbrady-techempower**
More php frameworks. I'll update symfony here as well, as I started it but it's not quite there yet. I also removed silex-orm in favor of silex. It was a much older version.

### CODE REVIEW

File: frameworks/PHP/kumbiaphp/bench/public/index.php [hunk 1]
```diff
@@ -54,15 +54,15 @@
...
+const APP_PATH = '/kumbiaphp/bench/app/';
 /**
  * Define el CORE_PATH.
  *
  * CORE_PATH:
  * - Ruta al directorio que contiene el núcleo de Kumbia (por defecto la ruta al directorio core)
  */
 //define('CORE_PATH', dirname(dirname(APP_PATH)).'/vendor/Kumbia/core/');
-const CORE_PATH = 'CORE_ROOT';
+const CORE_PATH = '/kumbiaphp/bench/app/vendor/Kumbia/core/';
```
Comment (joanhey) [inline frameworks/PHP/kumbiaphp/bench/public/index.php]: const CORE_PATH = '/kumbiaphp/vendor/Kumbia/core/';

File: frameworks/PHP/kumbiaphp/kumbiaphp-base.dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM tfb/php7:latest
+
+ADD ./ /kumbiaphp
+WORKDIR /kumbiaphp
+
+RUN git clone -b v1.0.0-rc.2 --single-branch --depth 1 https://github.com/KumbiaPHP/KumbiaPHP.git bench/app/vendor/Kumbia
+RUN git clone -b v0.4.0 --single-branch --depth 1 https://github.com/KumbiaPHP/ActiveRecord.git bench/app/vendor/Kumbia/ActiveRecord
```
Comment (joanhey) [inline frameworks/PHP/kumbiaphp/kumbiaphp-base.dockerfile]: RUN git clone -b v1.0.0-rc.2 --single-branch --depth 1 https://github.com/KumbiaPHP/KumbiaPHP.git vendor/Kumbia
RUN git clone -b v0.4.0 --single-branch --depth 1 https://github.com/KumbiaPHP/ActiveRecord.git vendor/Kumbia

**Comment by joanhey** [comment]
Later it's necessary to change the php.ini and php-fpm.conf.
The default is for development. And also change the errors log to stdout for check the errors.

:+1:  good job

**Comment by nbrady-techempower** [comment]
All PHP passing

**Pull request merged**